scripts/smp-server-digitalocean-droplet/files/opt/simplex/on_login.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+ip_address=$(hostname -I | awk '{print$1}')
+hash=$(cat /etc/opt/simplex/simplex.conf | grep hash: | cut -f2 -d":" | xargs)
```

Comment:
at some point I thought that it might be good to have hash as a parameter of .ini file so in addition to it all being in one location, the server would validate it.

@npoberezkin @efim-poberezkin

Reply:
But it's calculated in runtime, isn't it? Which means you'd have to dynamically edit the config file. Also, you can't supply the hash as a developer as part of configuration, and you cannot reuse config file after the hash has been added. I'm for storing the hash in a file, but not in the ini - let's just have a separate file where server writes hash on initialization, smth like `/etc/opt/simplex/pubkey_hash`?

Reply:
already done it, checking, commit soon